### ISSUE TITLE
Content Transfer to Association

### ISSUE THREAD

**Issue by mkanal** [eclipse-tractusx/sig-release#652]
## Description
Identification of content to be transferred to Association 

## Link 
- parent of https://github.com/eclipse-tractusx/item-relationship-service/issues/527
- parent of https://github.com/eclipse-tractusx/item-relationship-service/issues/371
- parent of https://github.com/eclipse-tractusx/item-relationship-service/issues/222

## Impact
-

## Additional information 
- (Consortia Outbound IP Clearance required)

- [ ] I'm willing to contribute to this feature

**Comment by stephanbcbauer** [comment]
was presented in open planning -> contributors and committer are involved

**Comment by stephanbcbauer** [comment]
Hello @jzbmw, @mkanal

Since the feature is a 24.08 feature and the development phase is coming to an end, we need a status on the feature.

- [ ] Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
- [ ] Please check whether the status (backlog, work in progress ...) is set correctly
- [ ] Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
- [ ] Please comment on the current status of the feature
- [ ] Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.

If you need any clarification, please get in touch, thank you very much.
Stephan

**Comment by stephanbcbauer** [comment]
Hello @jzbmw , @mkanal 
 
In a few weeks, we want to publish Tractus-X Release 24.08. For this, we need a clear status about the features (which will be included in the release and which will be cancelled or postponed to the next release).
 
Currently, the feature is still in status `Work in progress`. Please update the status by 09.07.24 eob.
 
If the feature is not ready for release 24.08, or it is possibly not planned at all, please set the status to `Not in release`. Please remember that in this case, any dependent components must be informed.
 
Thank you and best regards.

**Comment by mkanal** [comment]
Hello @stephanbcbauer 
feature will be closed after Content Transfer to Association and IP clearance is given. Still in scope of the release 24.8.

**Comment by mkanal** [comment]
IP Clearance is currently blocked by companies. I am still confident to resolve this in time 31.7.2024.
